frontend/src/v5/ui/routes/viewer/tickets/tickets.component.tsx

```diff
@@ -15,47 +15,89 @@
  */
 
+/* eslint-disable jsx-a11y/anchor-is-valid */
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/**
+ *  Copyright (C) 2022 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
 import TicketsIcon from '@mui/icons-material/FormatListBulleted';
 import { FormattedMessage } from 'react-intl';
 import { CardContainer, CardHeader } from '@/v5/ui/components/viewer/cards/card.styles';
-import { useEffect } from 'react';
-import { ViewerParams } from '@/v5/ui/routes/routes.constants';
-import { useParams } from 'react-router-dom';
-import { TicketsHooksSelectors } from '@/v5/services/selectorsHooks/ticketsSelectors.hooks';
-import { TicketsActionsDispatchers } from '@/v5/services/actionsDispatchers/ticketsActions.dispatchers';
-import { modelIsFederation } from '@/v5/store/tickets/tickets.helpers';
+
 import { CardContent } from '@/v5/ui/components/viewer/cards/cardContent.component';
-import { TicketsList } from './ticketsList/ticketsList.component';
-
-export const Tickets = () => {
-	const { teamspace, project, containerOrFederation } = useParams<ViewerParams>();
-	const isFederation = modelIsFederation(containerOrFederation);
-	const tickets = TicketsHooksSelectors.selectTickets(containerOrFederation);
-
-	useEffect(() => {
-		TicketsActionsDispatchers.fetchTickets(
-			teamspace,
-			project,
-			containerOrFederation,
-			isFederation,
-		);
-		TicketsActionsDispatchers.fetchTemplates(
-			teamspace,
-			project,
-			containerOrFederation,
-			isFederation,
-		);
-	}, [containerOrFederation]);
+import { CardContext, CardContextComponent, CardContextView } from '@components/viewer/cards/cardContext.component';
+import { useContext } from 'react';
+import { Button } from '@controls/button';
+import { TicketsCardViews } from './tickets.constants';
+import { TicketsListCard } from './ticketsList/ticketsListCard.component';
+
+export const TicketDetailCard = () => {
+	const constextValue = useContext(CardContext);
+
+	const goBack = () => {
+		constextValue.setView(TicketsCardViews.List);
+	};
+
+	return (
+		<CardContainer>
+			<CardHeader>
+				<TicketsIcon fontSize="small" />
+				<FormattedMessage id="viewer.cards.ticketsTitle" defaultMessage="Tickets" />
+				<Button onClick={goBack}>back</Button>
```

Comment:
I'm assuming this is just temporary, so I'll ignore the terrible styling and lack of formatting